How-to-access-Oppia-webpages.md

```diff
@@ -87,9 +88,21 @@ As you go through https://www.oppia.org/, you will find these entities.
 ### Other terms
 
 1. **Collection**: A group of explorations.
-2. **Community**: Anyone who teaches, learns, and contributes to any content on Oppia.
+2. - **Community** Anyone who teaches, learns, and contributes to any content on Oppia. Community members are respectful citizens whose main objective is to steward enriching learning opportunities.
 3. **Opportunity**: An opportunity is specific to an exploration, and it refers to a way any contributor can contribute content to Oppia.
 4. **Suggestion**: Any time any user contributes to Oppia, their contribution is added as a request which is known as a suggestion. There are two types of suggestions:
+5. **Category/Subject** A category or subject that anyone can learn about on Oppia. Learners can choose from a list of categories on the home page.
+6. **Contributor** Anyone who pitches in content for an exploration/lesson. One way that Oppia is more of a collaborative learning space, in contrast to more traditional online learning platforms.
+7. **Create** To start an exploration/lesson in order to teach a topic
```

Comment:
Let's drop this one as well; "create" is self-evident. Ditto for the next point.

Reply:
Dropped "create" definition, left the Contributor definition.